src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,23 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userInputToString = userInput.toString().split('');
+  const numberToGuessToString = numberToGuess.toString().split('');
```

Comment:
It's a good practice to ensure that the input parameters are of expected types before performing operations on them. Consider adding a type check for 'userInput' and 'numberToGuess' to ensure they are numbers and handle any potential type-related errors.